source/develop/release-management/features/network/detailed-host-network-qos.html.md

```diff
@@ -177,7 +177,7 @@ Since QoS API has been added in oVirt 3.5, it will be possible to expose Host Ne
 
 <b>Slightly out of date - should be similar, but other values will appear in the dialogs. A new category should appear under the DC/QoS subtab.</b>
 
-![ thumb | right](NetworkDialogQos.png  "fig: thumb | right") ![ thumb | right](InterfaceQosOverride.png  "fig: thumb | right")
+![ thumb | right](/images/wiki/NetworkDialogQos.png) ![ thumb | right](/images/wiki/InterfaceQosOverride.png)
```

Comment:
I think the ` thumb | right` isn't correct here. That's supposed to be for alt text, not directives. I'd suggest `Quality of Service network dialog screenshot` and `Quality of Service Override screenshot` as the alt text instead.